karma.conf-sauce.js

```diff
@@ -22,15 +23,22 @@ var sauceBrowsers = _.reduce([
   ['opera', '12'],
   ['opera', '11'],
 
+  ['android', '5'],
+  ['android', '4.4'],
   ['android', '4.3'],
   ['android', '4.0'],
 
-  ['safari', '8'],
+  ['safari', '8.0', 'OS X 10.10'],
```

Comment:
Do we have mobile safari as an option at all? To catch JIT bugs etc?

(I know there's a bug with Object.create and use strict that we might want to look out for)